src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form
+      action="#" 
+      class="form form--big" 
+      data-qa="big">
```

Comment:
```suggestion
      <form
        action="#" 
        class="form form--big" 
        data-qa="big"
      >
```
Consider fixing indents according to the code style guide